src/ng/compile.js

```diff
@@ -2455,6 +2469,24 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     }
 
 
+    function addEventBindingDirective(directives, eventName, isEventDirective) {
+      if (!isEventDirective) return;
+
+      directives.push({
+        compile: function(tElement, tAttr) {
```

Comment:
The main logic is in the linking function - should I be caching the parsed expression in the linking function then?